makefile.d/20-tools.mk

```
@@ -231,3 +231,15 @@ bin/typos-$(TYPOS_VERSION): | $(BIN_DIR)
 	@curl -sSfL 'https://github.com/crate-ci/typos/releases/download/v$(TYPOS_VERSION)/typos-v$(TYPOS_VERSION)-$(TYPOS_ARCH)-$(TYPOS_OSTYPE).tar.gz' --output - \
 	| tar xzvOf - './typos' > $@
 	@chmod +x $@
+
+####
+##  Go MultiMod Releaser
+####
+
+MULTIMOD_BIN := $(BIN_DIR)/multimod
+MULTIMOD_REPO_DIR := $(BIN_DIR)/opentelemetry-go-build-tools
+
+bin/multimod:
+	@if [ ! -d $(MULTIMOD_REPO_DIR) ]; then git clone git@github.com:open-telemetry/opentelemetry-go-build-tools.git --branch multimod/v0.13.0 $(MULTIMOD_REPO_DIR); fi
```

Using HTTPS works as ssh did not work for me by default
```suggestion
	@if [ ! -d $(MULTIMOD_REPO_DIR) ]; then git clone https://github.com/open-telemetry/opentelemetry-go-build-tools --branch multimod/v$(MULTIMOD_VERSION) $(MULTIMOD_REPO_DIR); fi
```

---

thanks for reviewing and for testing this 🙌